Fomo/AttractionType.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  AttractionType.swift
+//  Fomo
+//
+//  Created by Christian Deonier on 2/29/16.
+//  Copyright © 2016 TeamAwesome. All rights reserved.
+//
+
+import UIKit
+
+// Preferences are set by passing an array of AttractionTypes to endpoint
+class AttractionType: NSObject {
+    
+    var id: Int?
+    var name: String?
+    
```

Comment:
@jennify - is there a finite list from Google of the activity types that we could put in an enum. The User preference settings should align with how Google categorizes attractions